whylog/teacher/userintent.py

```diff
@@ -0,0 +1,20 @@
+class UserParserIntent(object):
+    def __init__(self, log_type_name, regex, primary_key_groups, data_conversions=None):
+        self.log_type_name = log_type_name
+        self.regex = regex
+        self.primary_key_groups = primary_key_groups
+        self.data_conversions = data_conversions
```

Comment:
This 'or' is cool! Normally I would write sth like "{} if data_conversions is None else data_conversions". Thanks!